### DISCUSSION TITLE
Welcome to e19-co227-digital-reporting-of-technical-visits Discussions!

### DISCUSSION THREAD

**Discussion by chandula00** [cepdnaclk/e19-co227-digital-reporting-of-technical-visits#16] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bimbara28** [comment, marked as answer]
Hi Chandula ! I'm Bimbara 👧

**Comment by Bimbara28** [comment]
Did you do that Calendar task?

**Reply by chandula00** [reply]
Yep 😁😃
